apps/src/music/views/MusicView.jsx

```diff
@@ -378,43 +387,45 @@ class UnconnectedMusicView extends React.Component {
             getLastMeasure: () => this.player.getLastMeasure()
           }}
         >
-          <div id="music-lab-container" className={moduleStyles.container}>
-            {this.state.showInstructions &&
-              instructionsPosition === InstructionsPositions.TOP &&
-              this.renderInstructions(InstructionsPositions.TOP)}
-
-            {this.state.timelineAtTop &&
-              this.renderTimelineArea(
-                true,
-                instructionsPosition === InstructionsPositions.RIGHT
-              )}
-
-            <div className={moduleStyles.middleArea}>
+          <PlayingContext.Provider value={{isPlaying: () => this.isPlaying()}}>
```

Comment:
I think we can further simplify this to just store a boolean `isPlaying` for the context value. That way we also guarantee that we re-render whenever the value changes automatically..

Reply:
Ah, I see.  Done.